sidebars.js

```diff
@@ -361,7 +361,8 @@ const sidebars = {
       type: "category",
       label: "Deploy a Rollup",
 			link: {
-				type: "generated-index",
+        id: "developers/rollup-overview",
```

Comment:
this should match the formatting for the rest of `sidebars.js` for this addition

Reply:
Thank you for your review. I will implement the changes requested. Could you clarify on this part though:

> this should match the formatting for the rest of `sidebars.js` for this addition